partiql-lang/src/main/kotlin/org/partiql/lang/planner/transforms/plan/PlanTyper.kt

```diff
@@ -227,7 +338,8 @@ internal object PlanTyper : PlanRewriter<PlanTyper.Context>() {
         return node.copy(
             input = input,
             common = node.common.copy(
-                typeEnv = typeEnv
+                typeEnv = typeEnv,
+                properties = input.getProperties()
```

Comment:
This change was for fixing the `ORDER BY` schema inference behavior.

Tests with corrected behavior: https://github.com/partiql/partiql-lang-kotlin/blob/e1582745db1fc79929d11ecd3d924e131d4bcf9c/partiql-lang/src/test/kotlin/org/partiql/lang/planner/transforms/PartiQLSchemaInferencerTests.kt#L658-L678.

Previously these tests would output a bag rather than a list.